packages/docs/src/demos/apollo/App.stories.js

```diff
@@ -61,36 +60,32 @@ const films = [
 ];
 
 export const MockedSuccess = MockTemplate.bind({});
-MockedSuccess.story = {
```

Comment:
Thank you for these updates! 
May I ask you to provision these changes in a separate pull request? That'd make it move evident while making this pull request scoped to the feature it introduces.

Reply:
Consider it done!

Reply:
done in #28

Reply:
Rebased! The PR changes are much better now :)